bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -467,7 +467,7 @@ spec:
             echo $(date -Iseconds) INFO: Failed to get ovnkube-master service from API. Retry "${retries}"/100 1>&2
             sleep 20
           done
-          if [ "${retries}" -ge 20 ]; then
+          if [ "${retries}" -ge 100 ]; then
```

Comment:
why are we changing the retries from 20 to 100? In sdn we use 20. Anyways let's see, now that the namespace part is fixed we should not be hitting this that often.

Reply:
Becase we retry 100 times;
          while [[ "${retries}" -lt 100 ]]; do
So we should fatal with the same amount. It should probably be a constant though

Reply:
Because we retry 100 times:
          while [[ "${retries}" -lt 100 ]]; do

This should have never been 20.